editor/project_converter_3_to_4.cpp

```diff
@@ -2860,9 +2859,9 @@ String ProjectConverter3To4::get_starting_space(const String &line) const {
 	return empty_space;
 }
 
-// Return object which execute specific function
-// e.g. in `var roman = kieliszek.funkcja()` to this function is passed everything before function which we want to check
-// so it is `var roman = kieliszek.` and this function return `kieliszek.`
+// Return object executing the function in the line.
```

Comment:
I would change this to “Returns the object that’s executing the function in the line.” Here’s why:

1. The comment above the declaration for `ProjectConverter3To4::get_starting_space` says “Returns”, not “Return”.
2. If the nouns “function” and “line” are going to use an article, then the noun “object” should too.
3. It’s often OK to leave out the word “that”. I don’t think that it’s ever correct to leave out the word “that’s”.